src/images.js

```diff
@@ -0,0 +1,17 @@
+import image1 from './assets/ca.jpg';
+import image2 from './assets/dog.jpg';
+import image3 from './assets/lisa.jpg';
+import image4 from './assets/vedmed.jpg';
+import image5 from './assets/volk.jpg';
+import image6 from './assets/panda.png';
+
+const images = {
+  'Кіт': image1,
+  'Песик': image2,
+  'Лисиця': image3,
+  'Ведмідь': image4,
+  'Вовк': image5,
+  'Панда': image6
```

Comment:
Don't use Ukrainian for any object keys, only English

Reply:
everything is here src\data\wordObjects.js